pyveg/src/data_analysis_utils.py

```diff
@@ -839,6 +839,80 @@ def resample_time_series(df, col_name="offset50", period="D"):
     return new_series
 
 
+def variance_moving_average_time_series(df, col_name="offset50", lenght=1):
```

Comment:
"length" is misspelled as "lenght" here (but it also is in the function itself so should still work!  (i.e. if we fix one, need to fix both...)